src/EditorFeatures/TestUtilities/ExtractInterface/TestExtractInterfaceOptions.cs

```diff
@@ -44,7 +45,8 @@ public TestExtractInterfaceOptionsService()
             List<string> conflictingTypeNames,
             string defaultNamespace,
             string generatedNameTypeParameterSuffix,
-            string languageName)
+            string languageName,
+            CancellationToken cancellationToken)
```

Comment:
unused?

Reply:
Yeah, it's test code. Method is implementing an interface.